src/main/java/io/supertokens/config/Config.java

```diff
@@ -110,16 +116,32 @@ public static void loadAllTenantConfig(Main main, TenantConfig[] tenants)
         // At this point, we know that all configs are valid.
```

Comment:
verify that if the base tenant of an app's config has changed, then we also reload all the sub tenants (if required)

Reply:
Done